.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -118,6 +161,66 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.destination-slot-name }}
 
+      - name: Checkout Code Repository
+        if: inputs.app-code-repository != ''
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          repository: ${{ inputs.app-code-repository }}
+          ref: ${{ inputs.app-code-repository-ref }}
+          token: ${{ secrets.CODE_REPOSITORY_WRITE_TOKEN }}
+
+      - name: Remove Old Tags
+        uses: Lombiq/GitHub-Actions/.github/actions/remove-old-tags@issue/OSOE-861
+        with:
+          tag-prefix: ${{ inputs.tag-prefix }}
+
+      - name: Delete Old prefix/latest Tag
+        run: |
+            $latestTag = (git tag --list '${{ inputs.tag-prefix }}/latest')
+            if ($latestTag)
+            {
+                git tag -d '${{ inputs.tag-prefix }}/latest'
+                git push origin ':refs/tags/${{ inputs.tag-prefix }}/latest'
+            }
+
+      - name: Move Latest Tag (Swap) and Add Timestamp Tag
+        if: inputs.app-code-repository != ''
+        run: |
+            if([string]::IsNullOrEmpty('${{ inputs.tag-prefix }}'))
+            {
+                throw 'Tag prefix not set, exiting'
```

Comment:
I get this when doing a swap. I only added the `app-code-repository` parameter and the `CODE_REPOSITORY_WRITE_TOKEN` secret to the previously existing workflow, but I'd like to use the defaults of the prefixes.

Reply:
I've pushed a fix for this now, please run the workflow again whenever you get a chance.

Reply:
This is indeed fixed, thanks.